.travis.yml

```diff
@@ -15,12 +15,16 @@ env:
 services:
   - docker
 
+before_install:
+    - pip install pep8
+
 install:
   - docker build -t ${DOCKER_REPOSITORY}:${TRAVIS_COMMIT} .
 
 script:
   - docker run ${DOCKER_REPOSITORY}:${TRAVIS_COMMIT} /bin/bash -c "./scripts/build.py -t"
   - docker run ${DOCKER_REPOSITORY}:${TRAVIS_COMMIT} /bin/bash -c "./scripts/build.py -t --python-path python3"
+  - find ffig scripts tests -name \*.py -exec pep8 {} +
```

Comment:
maybe use `git ls-files | grep -e ".*\.py"` here as the folder list will need manually updating.